client/src/pages/EditProfile.js

```diff
@@ -21,15 +21,31 @@ const initalState = {
   description: "",
   disabled: true,
   snackbaropen: true,
-  snackbarmsg: "test"
+  snackbarmsg: "test",
+  edit: "0",
+  formChanges: false
 };
 
 class EditProfilePage extends Component {
+  wowState = {
+    firstName: "",
+    lastName: "",
+    gender: "",
+    birthDate: "",
+    phone: "",
+    address: "",
+    description: "",
+    disabled: true,
+    snackbaropen: true,
+    snackbarmsg: "test",
+    edit: "0",
+    formChanges: false
+  };
```

Comment:
I don't understand why we keep changing each other's code. Who was initially responsible for this page?

Reply:
@Kartihan-sri was responsible for the editProfile page